tests/js/jest-setup.js

```diff
@@ -17,6 +17,7 @@ beforeEach(() => {
 })
 
 afterEach(() => {
+	global.__PAYMENT_METHOD_FEES_ENABLED = false;
```

Comment:
Each test has the ability to overwrite the value of the flag to test the different behavior - resetting its default value here